docs/getting-started/configuration.md

```diff
@@ -0,0 +1,154 @@
+---
```

Comment:
After reading this document, I don't know whether this configuration is only for `sb run` or not. If yes, why do you recommend `sub execute`?

Reply:
do not have scenarios for exec yet